src/main/java/xyz/groundx/caver_ext_kas/kas/kip7/KIP7.java

```diff
@@ -0,0 +1,1459 @@
+     * @return Kip7TransactionStatusResponse
+     * @throws ApiException
+     */
+    public Kip7TransactionStatusResponse approve(String addressOrAlias, String owner, String spender, BigInteger amount) throws ApiException {
+        return approve(addressOrAlias, owner, spender, Numeric.toHexStringWithPrefix(amount));
+    }
+
+    /**
+     * Authorizes the `spender` to send a certain amount of tokens on behalf of the `owner` asynchronously. <br>
+     * POST /v1/contract/{contract-address-or-alias}/approve <br>
+     *
+     * <pre>Example :
+     * {@code
+     * ApiCallback<Kip7TransactionStatusResponse> callback = new ApiCallback<Kip7TransactionStatusResponse>() {
+     *     ....implements callback method.
+     * }
+     *
+     * String contractAlias = "";
+     * String owner = "";
+     * String spender = "";
+     * BigInteger amount = BigInteger.valueOf(10).multiply(BigInteger.TEN.pow(18)); // 10 * 10^18
+     *
+     * caver.kas.kip7.approveAsync(contractAlias, owner, spender, amount, callback);
+     *
+     * }
+     * </pre>
+     *
+     * @param addressOrAlias Contract address (in hexadecimal with the 0x prefix) or an alias.
+     * @param owner Klaytn account address of the owner delegating the token tranfer. The default is the contract deployer account.
+     * @param spender Klaytn account address to approve delegated token transfer.
+     * @param amount Approved token amount.
+     * @param callback The callback to handle response
+     * @return Call
+     * @throws ApiException
+     */
+    public Call approveAsync(String addressOrAlias, String owner, String spender, BigInteger amount, ApiCallback<Kip7TransactionStatusResponse> callback) throws ApiException {
+        return approveAsync(addressOrAlias, owner, spender, Numeric.toHexStringWithPrefix(amount), callback);
+    }
+
+    /**
+     * Authorizes the `spender` to send a certain amount of tokens on behalf of the `owner`. <br>
+     * POST /v1/contract/{contract-address-or-alias}/approve <br>
+     *
+     * <pre>Example :
+     * {@code
+     * String contractAlias = "";
+     * String owner = "";
+     * String spender = "";
+     * String amount = "0x8ac7230489e80000"; // 10 * 10^18
+     *
+     * Kip7TransactionStatusResponse response = caver.kas.kip7.approve(contractAlias, owner, spender, amount);
+     * }
+     * </pre>
+     *
+     * @param addressOrAlias Contract address (in hexadecimal with the 0x prefix) or an alias.
+     * @param owner Klaytn account address of the owner delegating the token tranfer. The default is the contract deployer account.
+     * @param spender Klaytn account address to approve delegated token transfer.
+     * @param amount Approved token amount. (in hexadecimal with the 0x prefix)
+     * @return Kip7TransactionStatusResponse
+     * @throws ApiException
+     */
+    public Kip7TransactionStatusResponse approve(String addressOrAlias, String owner, String spender, String amount) throws ApiException {
```

Comment:
Can you implement `public Kip7TransactionStatusResponse approve(String addressOrAlias, String spender, String amount)` function which does not receive `owner` parameter? The default is the contract deployer account (In yaml file, `ApproveKip7TokenRequest` component has info for this).

Reply:
I added it.